src/components/SignIn.tsx

```diff
@@ -283,6 +284,14 @@ const SignIn = ({ onWeb3ProviderSet, onWeb3AuthInstanceSet }: SignInProps) => {
     /* eslint-disable-next-line */
   }, []);
 
+  const { isConnected } = useAccount()
+  const {data: signer} = useSigner();
```

Comment:
```suggestion
  const { data: signer } = useSigner();
```

Reply:
Have you configured format on save in vscode to prettier yet? It should catch things like this